packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -31,7 +31,7 @@ export type AutocompleteListProps<T> = {
   limitTags?: number;
   inputValue?: string;
   clearText?: string;
-  value?: T extends unknown[] ? T : T[];
+  value?: any;
```

Comment:
yeah. The suggestion was from [this comment](https://github.com/openmsupply/openmsupply-client/pull/1110#discussion_r835968146) but given that I didn't really follow the syntax and it wasn't allowing the typing to work, I've abandoned.